.github/workflows/test.yml

```
@@ -21,6 +21,9 @@ jobs:
           path: venv
           key: venv-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('**/requirements*.txt') }}
 
+      - name: Install developer tools
+        run: make bootstrap
+
       - name: Install dependencies
         run: make requirements-dev
```

```suggestion
        run: invoke requirements-dev
```
likewise for line 31 (I can't add as a suggestion...) which should be replaced for
```
        run: invoke test
```